core/src/accounts_hash_verifier.rs

```diff
@@ -38,6 +40,7 @@ impl AccountsHashVerifier {
         fault_injection_rate_slots: u64,
         snapshot_interval_slots: u64,
     ) -> Self {
+        let thread_pool = accounts_db::make_min_priority_thread_pool();
```

Comment:
Can we just create it with the flag?

Reply:
```
let thread_pool = if package.test_hash {
    Some(accounts_db::make_min_priority_thread_pool())
} else {
    None
};
```

Reply:
this has been pulled into a separate pr. #15149